server_admin/topics/events/admin.adoc

```diff
@@ -1,36 +1,37 @@
 
 === Admin Events
 
-Any action an admin performs within the admin console can be recorded for auditing purposes.
-The Admin Console performs administrative functions by invoking on the {project_name} REST interface.  {project_name}
-audits these REST invocations.  The resulting events can then be viewed in the Admin Console.
+The Admin Console performs its functions by invoking the {project_name} REST interface. Because {project_name} audits all REST invocations, {project_name} can record all administrator actions within the Admin Console. You can view the action events in the Admin Console.
 
```

Comment:
How about 
"You can record all actions that are performed by an administrator in the Admin Console.  The Admin Console performs administrative actions by invoking the {project_name} REST interface and {project_name} audits these REST invocations. You can view the resulting events in the Admin Console."